lms/djangoapps/verify_student/views.py

```diff
@@ -924,12 +922,15 @@ def _validate_parameters(self, request, has_initial_verification):
             return None, HttpResponseBadRequest(msg)
 
         # If provided, parse the course key and checkpoint location
+        # As it's not currently used in the new flow for submission, only log a warning if it's invalid
```

Comment:
Actually, I changed my mind.
Why don't we remove this if condition all together? It's not really used in the submit-photos endpoint.